### ISSUE TITLE
Screen reader inaccessible column controls

### ISSUE THREAD

**Issue by aardrian** [ag-grid/ag-grid#3341]
**I'm submitting a ...**  (check one with "x")
```
[X] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of ag-Grid'
[] support request => see 'Requesting Community Support'
```

**Customers of ag-Grid**

I am not a customer, but I am evaluating ag-Grid for a client, a potential ag-Grid customer.

**Requesting Community Support**

I understand this will be closed if inactive for one month. However, as a non-customer I have no other way to report bugs.

**Providing a Reproducible Scenario**

1. Go to the [ag-Grid page promoting its accessibility](https://www.ag-grid.com/javascript-grid-accessibility/).

2. Scroll to the bottom and find the only demo on the page, [opening the Plunker in its own window](https://www.ag-grid.com/example-runner/vanilla.php?section=javascript-grid-accessibility&example=accessibility&enterprise=1&generated=1&grid=%7B%22theme%22%3A%22ag-theme-balham%22%2C%22noStyle%22%3A0%2C%22height%22%3A%22100%25%22%2C%22width%22%3A%22100%25%22%2C%22enterprise%22%3A1%7D) (I provide that link for ease of accessing it, but still do step 1 so you have the latest ag-Grid-sanctioned accessible demo).

3. Using a screen reader, navigate to the column header for any column.

4. Attempt to find or activate the hamburger control that allows you to pin a column, size a column, reset a column, or filter a column.

**Current behavior**
No hamburger controls are announced nor can they be accessed.

**Expected behavior**
The hamburger control should announce as a [disclosure widget](https://www.w3.org/TR/wai-aria-practices-1.1/#disclosure) with an accessible name describing it.

**Please tell us about your environment:**

* **ag-Grid version:** Whatever is the version in your live demo.

* **Browser:**
  * Windows 10, Internet Explorer 11, JAWS 2018
  * Windows 10, Chrome 76, JAWS 2018
  * Windows 10, Firefox 69, NVDA 2019.2

* **Language:** Whatever is the language in your live demo.

**Comment by davis4acca** [comment]
Hi,

Your ticket has been flagged as in_zendesk, it means that we recognize this as an issue worth investigating so we have moved it to our official support channel for customers (zendesk)

We will carry on there with the investigation and we will update you as soon as possible.

Thanks

**Comment by davis4acca** [comment]
Hi,

Thank you for your email. We are aware of this issue and have a feature request in our backlog:

You can follow the status of feature requests, bugs and releases using our pipeline:
https://www.ag-grid.com/ag-grid-pipeline/

If you are an ag-Grid Enterprise Customer, please raise this in our Zendesk support system - contact info@ag-grid.com for access. This is our primary channel for Support.



This is the reference and summary of the relevant issue:

AG-3223: ARIA / Accessibility / Keyboard navigation allow user to use filter and filter options by navigating with keyboard


AG-2629: Accessibility - Allow screen readers to access the column headers to apply sort and filtering capabilities


AG-3253: Accessibility / navigation Allow to navigate with keyboard though the context menu when it gets opened
